### DISCUSSION TITLE
安卓局域网代理

### DISCUSSION THREAD

**Discussion by zz0133zz** [2dust/v2rayNG#3227] [General]
怎么开启安卓局域网代理，让其他处于同一局域网下的设备访问外网，求解

**Comment by fishpoaem** [comment]
> 怎么开启安卓局域网代理，让其他处于同一局域网下的设备访问外网，求解

左上角三横菜单-->设置-->进阶设置-->允许局域网连接----勾选上，然后再到其他局域网设备上配置代理
